### ISSUE TITLE
Combine Views

### ISSUE THREAD

**Issue by smuaq** [twinssbc/Ionic-Calendar#155]
Is there a way to combine the views & show them together? Like I want to show the day view with month view? 

Thanks for the great calendar by the way!

**Comment by twinssbc** [comment]
@smuaq It does not support display month view and day view at the same time. Different people may have different intention to display the events on a calendar. I chose the most common way to implement this calendar.

**Comment by smuaq** [comment]
Right, thanks for your reply. I was looking to implement it in way that shows both week view along with the selected day's day view. I guess i'll have to customize it.
